README.md

```diff
@@ -1,9 +1,10 @@
-[![CircleCI](https://img.shields.io/circleci/project/github/3box/3box-js/master.svg?style=for-the-badge)](https://circleci.com/gh/3box/3box-js)
+[![CircleCI](https://img.shields.io/circleci/project/github/3box/3box-js.svg?style=for-the-badge)](https://circleci.com/gh/3box/3box-js)
```

Comment:
removed specific branch? want develop or master? and would need same change in readme template

Reply:
I'm pretty sure I just generated from the template. It shouldn't be master, and I think not specifying gives the default one.

Reply:
ah yeah I see now in template, yeah not specifying is develop, 👍